File: Application/StatsApi/Models/PlayerGameStats.cs
namespace StatsApi.Models;

public class PlayerGameStats
{
    public Game? Game { get; set; }
    public Player? Player { get; set; }

    public int? Points { get; set; }
    public int? Made1p { get; set; }
    public int? Miss1p { get; set; }
    public int? Made2p { get; set; }
    public int? Miss2p { get; set; }
    public int? Made3p { get; set; }
    public int? Miss3p { get; set; }
    public int? Assists { get; set; }
    public int? Rebounds { get; set; }
    public int? OffensiveRebounds { get; set; }
    public int? DefensiveRebounds { get; set; }
    public int? Steals { get; set; }
    public int? Turnovers { get; set; }
    public int? Pir { get; set; }
}

File: app/Statistics/frontend/src/models/PlayerGameStats.ts
import type { Player } from "./Player";

export type PlayerGameStats = {
  id: string;
  game: string;
  player: Player;
  points: number;
  made1p: number;
  miss1p: number;
  made2p: number;
  miss2p: number;
  made3p: number;
  miss3p: number;
  assists: number;
  rebounds: number;
  offensiveRebounds: number;
  defensiveRebounds: number;
  steals: number;
  turnovers: number;
  pir: number;
  personalFouls: number;
  recievedFouls: number;
  blocks: number;
  recievedBlocks: number;
  technicalFouls: number;
  secondsPlayed: number;
  isStarter: boolean;
};
